### PR TITLE
Watch3D in customizer

### PR DESCRIPTION
**Pull request by aosyatnik**
### Purpose

Link to YouTrack:
[MAGN-8962](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8962) Enable publish Watch3D node with Customizer

This series of PRs enables next functionality:
1. Ability to store some workspace specific information, such as X-offset, Y-Offset, Zoom of node graph view (part of workspace, where only nodes are presented.) 
2. Ability to store camera data of Watch3D nodes.

In this PR I add `Watch3DCamera`, the class that is representation of Helix Camera, used in Watch 3D node.

Flood PR: https://github.com/DynamoDS/Flood/pull/158
Reach PR: https://github.com/DynamoDS/Reach/pull/132
### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.

### Reviewers

@pboyer 
@marimano 
@pbidenko 

### FYIs

@jnealb

### CODE REVIEW

File: src/VisualizationTests/WpfVisualizationTests.csproj [hunk 1]
```diff
@@ -178,6 +178,7 @@
     <ProjectReference Include="..\Libraries\Watch3DNodeModels\Watch3DNodeModels.csproj">
       <Project>{31183026-DE70-49CB-BC7C-0DFD0A088F62}</Project>
       <Name>Watch3DNodeModels</Name>
+      <Private>False</Private>
```
Comment (aosyatnik) [inline src/VisualizationTests/WpfVisualizationTests.csproj]: Do not copy Watch3DNodeModels.dll in root Dynamo folder.

File: src/Libraries/Watch3DNodeModels/Watch3D.cs [hunk 2]
```diff
@@ -16,6 +16,45 @@
 
 namespace Watch3DNodeModels
 {
+    public class Watch3DCamera
+    {
+        private const string DefaultCameraName = "Background Preview";
+
+        public string Name { get; set; }
```
Comment (ramramps) [inline src/Libraries/Watch3DNodeModels/Watch3D.cs]: Comments?

**Comment by ramramps** [comment]
@aosyatnik - Add proper comments and then LGTM

**Comment by Racel** [comment]
@ramramps - Please take a look. There are 2 other PRs related to this in Flood and in Reach.

https://github.com/DynamoDS/Flood/pull/158
https://github.com/DynamoDS/Reach/pull/132

**Comment by ramramps** [comment]
@aosyatnik - LGTM

**Pull request merged**